### PR TITLE
Add Medical Surgery Kits and give them to MDs/CMO

### PR DESCRIPTION
**Pull request by redmoogle**
# General Documentation

### Intent of your Pull Request

Adam requested me to atomize his PR.

This PR adds a medical kit(which is basically the normal one but with surgery tools added) and gives to MDs/CMOs

### Why is this change good for the game?

Field Tendwounds, and should help encourage less dragging _hopefully_

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
MDs/CMOs get surgery tools ontop of their normal box

### What general grouping does this PR fall under? 
Medical Rebalance/tweak

# Changelog
:cl:  
rscadd: MDs/CMOs now spawn with a surgery kit
/:cl:

### CODE REVIEW

**Comment by Hopekz** [comment]
Neat no weird sprites

**Comment by JamieD1** [comment]
On Hold until the other change is finished - Cargo im not having too many systems changing at once.

**Review by JamieD1 — APPROVED** [review]
